src/persist-client/examples/admin.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! Commands for read-write administration of persist state
+
+use mz_build_info::BuildInfo;
+use mz_ore::metrics::MetricsRegistry;
+use mz_ore::now::SYSTEM_TIME;
+use mz_persist_client::{PersistConfig, ShardId};
+
+use prometheus::proto::{MetricFamily, MetricType};
+use std::str::FromStr;
+use tracing::info;
+
+use crate::inspect::StateArgs;
+
+/// Commands for read-write administration of persist state
+#[derive(Debug, clap::Args)]
+pub struct AdminArgs {
+    #[clap(subcommand)]
+    command: Command,
+}
```

Comment:
let's make `read_only` one of the stock AdminArgs arguments, and have it default to `true`

Reply:
you're about to find out lovely things about clap (bool that defaults to true is not possible 🚮). love the idea though, done!